### PR TITLE
refactor(sharepoint.information): use oui-tile oui-progress

### PR DESCRIPTION
**Pull request by TedKac**

### CODE REVIEW

File: src/sharepoint/information/INFORMATION.html [hunk 1]
```diff
@@ -6,83 +6,69 @@
...
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'sharepoint_dashboard_access' | translate}}">
+                        <oui-tile-definition data-term="{{:: 'sharepoint_dashboard_url_label' | translate}}">
+                            <oui-tile-description>
+                                <a data-ng-href="https://{{informationsCtrl.sharepoint.url}}" 
+                                   target="_blank"
+                                   data-ng-if="informationsCtrl.sharepoint.state !== 'creating'">
+                                    <span data-ng-bind="'https://' + informationsCtrl.sharepoint.url"></span>
+                                    <span class="fa fa-external-link" aria-hidden="true"></span>
```
Comment (FredericEspiau) [inline src/sharepoint/information/INFORMATION.html]: Alternative text for screen-reader users

File: src/sharepoint/information/INFORMATION.html [hunk 2]
```diff
@@ -6,83 +6,69 @@
...
-                                </li>
-                            </ul>
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'sharepoint_dashboard_access' | translate}}">
+                        <oui-tile-definition data-term="{{:: 'sharepoint_dashboard_url_label' | translate}}">
+                            <oui-tile-description>
+                                <a data-ng-href="https://{{informationsCtrl.sharepoint.url}}" 
+                                   target="_blank"
+                                   data-ng-if="informationsCtrl.sharepoint.state !== 'creating'">
```
Comment (FredericEspiau) [inline src/sharepoint/information/INFORMATION.html]:
```suggestion
                                   data-ng-if="informationsCtrl.sharepoint.state !== 'creating'" rel="noopener">
```

File: src/sharepoint/information/INFORMATION.html [hunk 3]
```diff
@@ -6,83 +6,69 @@
...
+                                    <span data-ng-bind="'https://' + informationsCtrl.sharepoint.url"></span>
+                                    <span class="fa fa-external-link" aria-hidden="true"></span>
+                                </a>
+                                <span data-translate="sharepoint_dashboard_creating_url_label"
+                                      data-ng-show="informationsCtrl.sharepoint.state === 'creating'"></span>
```
Comment (FredericEspiau) [inline src/sharepoint/information/INFORMATION.html]:
```suggestion
                                      data-ng-if="informationsCtrl.sharepoint.state === 'creating'"></span>
```

File: src/sharepoint/information/INFORMATION.html [hunk 4]
```diff
@@ -6,83 +6,69 @@
...
+                        </oui-tile-definition>
+                        <oui-tile-definition data-term="{{:: 'sharepoint_dashboard_linked_exchange_label' | translate}}"
+                                             data-ng-if="informationsCtrl.associatedExchange">
+                            <oui-tile-description>
+                                <span data-ng-bind="SharepointCtrl.exchangeId"
+                                      data-ng-if="!informationsCtrl.associatedExchangeLink"></span>
+                                <a data-ng-href="{{informationsCtrl.associatedExchangeLink}}"
+                                   data-ng-bind="SharepointCtrl.exchangeId"
+                                   data-ng-click="informationsCtrl.setExchange()"
+                                   data-ng-if="informationsCtrl.associatedExchangeLink">
```
Comment (FredericEspiau) [inline src/sharepoint/information/INFORMATION.html]:
```suggestion
                                   data-ng-if="informationsCtrl.associatedExchangeLink"></a>
```

File: src/sharepoint/information/INFORMATION.html [hunk 5]
```diff
@@ -6,83 +6,69 @@
...
-                                </dl>
-                            </div>
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'sharepoint_dashboard_quota' | translate}}">
+                        <oui-tile-definition>
+                            <oui-tile-description>
+                                <oui-progress data-min-value="0"
+                                              data-max-value="1024">
+                                    <oui-progress-bar data-ng-if="informationsCtrl.sharepoint.currentUsage/1024/1024/1024 <= 70"
```
Comment (FredericEspiau) [inline src/sharepoint/information/INFORMATION.html]: Hard to read, `informationsCtrl.sharepoint.currentUsageInGb <= 70` makes more sense

Comment (FredericEspiau) [inline src/sharepoint/information/INFORMATION.html]: Ok I understand what you wanted to do :). What you suggested this morning felt easier :

```js
<oui-progress-bar data-type="{{ $ctrl.ouiProgressBarType }}" ...
```

Comment (TedKac) [reply src/sharepoint/information/INFORMATION.html]: In fact, attribute type in oui-progress-bar is a one-time binding attribute, don't think we could use this tricks.

File: src/sharepoint/information/INFORMATION.html (hunk 1 above)
Comment (FredericEspiau) [reply src/sharepoint/information/INFORMATION.html]:
```js
<span data-translate="common_open_new_tab" class="sr-only"></span>
```

Comment (TedKac) [reply src/sharepoint/information/INFORMATION.html]: Fixed 😄

File: src/sharepoint/information/INFORMATION.html (hunk 5 above)
Comment (TedKac) [reply src/sharepoint/information/INFORMATION.html]: done 😄

File: src/sharepoint/information/sharepoint-information.controller.js [hunk 6]
```diff
@@ -73,4 +73,8 @@ angular
     setExchange() {
       this.productsService.setSelectedProduct(this.associatedExchange);
     }
+
+    currentusageInGb() {
+      return this.sharepoint.currentUsage/(1024 ** 3);
```
Comment (antleblanc) [inline src/sharepoint/information/sharepoint-information.controller.js]:
```suggestion
      return this.sharepoint.currentUsage / (1024 ** 3);
```

**Comment by jleveugle** [comment]
Hello @TedKac,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @TedKac,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by TedKac** [comment]
Conflicts fixed 😄

**Pull request merged**